WordPress/Classes/ViewRelated/Post/Revisions/Browser/RevisionBrowserState.swift

```diff
@@ -0,0 +1,26 @@
+class RevisionBrowserState {
+    typealias RevisionSelectedBlock = (Revision) -> Void
+
+    let revisions: [Revision]
+    var currentIndex: Int
+    var onRevisionSelected: RevisionSelectedBlock
```

Comment:
Consider making this `private(set)`, since this it not changed once the state is created.  It's minor but it would ensure the closure is not replaced by mistake from outside code.